src/views/contact.hbs

```diff
@@ -42,18 +43,13 @@
         <input type="checkbox" class="form-check-input" id="contact-check" required>
         <label class="form-check-label" for="contact-check">
           Hiermit bestätige ich meine Übereinstimmung mit <a href="#" data-toggle="modal"
-            data-target="#termsModal">Terms
-            &amp; Conditions</a> und <a href="#" data-toggle="modal" data-target="#privacyModal">Privacy Policy</a>.
+            data-target="#termsModal">Urheberrecht</a> und <a href="#" data-toggle="modal"
+            data-target="#privacyModal">Datenschutzerklärung</a>.
```

Comment:
@hikes101 please, check the German grammar :)

Reply:
"Hiermit bestätige ich meine Übereinstimmung mit **dem** Urheberrecht und **der** Datenschutzerklärung."